cilantro_audit/auditor_page.py

```diff
@@ -1,27 +1,64 @@
-import kivy
-from kivy.lang import Builder
-from kivy.uix.screenmanager import Screen
+from cilantro_audit import globals
 
-from cilantro_audit.constants import KIVY_REQUIRED_VERSION
+from kivy.app import App
+from kivy.uix.screenmanager import Screen
 
-kivy.require(KIVY_REQUIRED_VERSION)
+from cilantro_audit.constants import RGB_GREEN
 
-auditorPageLayout = Builder.load_file('./widgets/auditor_page.kv')
+from cilantro_audit.templates.cilantro_navigator import CilantroNavigator
+from cilantro_audit.templates.cilantro_button import CilantroButton
+from cilantro_audit.templates.cilantro_label import CilantroLabel
 
 
 class AuditorPage(Screen):
-    def build(self):
-        return auditorPageLayout
+    def __init__(self, **kw):
+        super().__init__(**kw)
+        self.populate_page()
+
+    def populate_page(self):
+        self.clear_widgets()
+        template_page = CilantroNavigator()
+
+        template_page.header_title.clear_widgets()
+        template_page.header_title.add_widget(CilantroLabel(text='Auditor Page',
+                                                            color=RGB_GREEN))
+
+        template_page.body_nav_btns.add_widget(CilantroButton(text='Submit New Audit',
+                                                              size_hint_y=None,
+                                                              height=70,
+                                                              on_release=submit_new_audit))
+        template_page.body_nav_btns.add_widget(CilantroButton(text='View Submitted Audits',
+                                                              size_hint_y=None,
+                                                              height=70,
+                                                              on_release=view_submitted_audits_page))
+
+        template_page.footer_logout.bind(on_release=lambda _: logout())
+
+        self.add_widget(template_page)
 
-    def submit_new_audit(self):
-        return
 
-    def view_submitted_audits(self):
-        return
+class CDF(Screen):
```

Comment:
Not sure what this does. Commenting it out has no effect on the app. Could you explain it?